src/convert.jl

```diff
@@ -130,6 +130,32 @@ function convert(::Type{VPolygon}, P::AbstractPolytope)
     return VPolygon(vertices_list(P))
 end
 
+"""
+    convert(::Type{VPolygon}, X::LazySet)
+
+Generic conversion to polygon in vertex representation.
+
+### Input
+
+- `type` -- target type
+- `X`    -- set
+
+### Output
+
+The 2D set represented as a polygon.
+
+### Algorithm
+
+First we concretize `X` obtaining a (possibly) new representation `Xc`; then we
+wrap the result into a polygon in vertex representation by computing the list of
+vertices of `Xc`. The vertices of the resulting polygon in vertex representation
+are guaranteed to be sorted in counter-clockwise fashion.
```

Comment:
The docstring is outdated.
```suggestion
We compute the list of vertices of `X` and wrap the result in a polygon in
vertex representation, which guarantees that the vertices are sorted in
counter-clockwise fashion.
```